client/src/components/Workflow/TrsImport.vue

```diff
@@ -105,9 +105,20 @@ export default {
                     this.trsTool = tool;
                     this.errorMessage = null;
                     if (this.isAutoImport) {
-                        const version = this.trsTool.versions.find((version) => version.id === this.queryTrsVersionId);
+                        /* Resolve discrepancy between workflowhub, which sends an id as query parameter,
+                           and dockstore, which uses the version name as the query parameter.
+                           Should just be one of them eventually. */
+                        let versionField = "name";
+                        const version = this.trsTool.versions.find((version) => {
+                            if (version.name == this.queryTrsVersionId) {
```

Comment:
I get a warning on my IDE to use `===` instead of `==`, but I guess it doesn't matter :sweat_smile:

Reply:
Yeah, meant to do triple, I'll push a fix